test/Npgsql.Tests/MultipleHostsTests.cs

```diff
@@ -20,7 +20,6 @@
 
 namespace Npgsql.Tests;
 
-[NonParallelizable]
```

Comment:
Nice that this works...

Reply:
Pretty sure that's because you've made `DatabaseStateInfo` bound to a data source, and we've started using data sources in tests.

Reply:
Yep